docs/docs/installation_guide/installation/pre-installation/china-prerequisites.md

```diff
@@ -0,0 +1,131 @@
+---
+id: china-prerequisites
+title: China Regions Prerequisites
+sidebar_label: China Regions Prerequisites
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section is just for China regions(Beijing, Ningxia and Hongkong), The initial prerequisites include preparing an ICP licensed domain, creating OIDC IdP client and creating Service Workbench root user in OIDC IdP.
+
+## Preparing an ICP licensed domain
+
+Prepare an [ICP][icp] licensed domain. The Service Workbench console is served via CloudFront distribution which is considered as an Internet information service. It is referred to as **Service Workbench Console Domain**.
+For Hongkong region, this part is not mandatory.
+ 
+## Creating OIDC IdP Client
+
+[Cognito User Pool][cognito] is not supported in AWS China Regions. If you choose to deploy the solution in AWS China Regions, an OpenID Connector (OIDC) IdP client is required.
+
+You can use different kinds of OpenID Connector(OIDC) providers. This section introduces Option 1, Option 2 and Option 3.
+
+- (Option 1) [Authing][authing] is an example of a third-party authentication provider. 
+- (Option 2) [Keycloak](https://github.com/aws-samples/keycloak-on-aws) is a solution maintained by AWS and can serve as an authentication identity provider.
```

Comment:
Keycloak on AWS, link to solution official page is better

Reply:
Yes, changed to solution official page